src/stylesheets/components/_tag.scss

```diff
@@ -0,0 +1,22 @@
+.app-c-tag--status {
+  @include govuk-font-regular-16;
+  display: block;
+  line-height: 1;
+}
+
+.app-c-tag--status {
+  a:link,
+  a:visited,
+  a:active,
+  a:hover {
+    color: $govuk-white;
+  }
+}
+
+.app-c-tag--status a:hover {
+  background-color: $govuk-light-blue;
+}
+
+a.app-c-tag--status:focus {
```

Comment:
Should this be `.app-c-tag--status a:focus`?

Reply:
This is now deleted